src/bosh-director/lib/bosh/director/jobs/delete_vm.rb

```diff
@@ -17,23 +17,31 @@ def initialize(vm_cid)
 
       def perform
         logger.info("deleting vm: #{@vm_cid}")
-        begin
-          instance = Bosh::Director::Api::InstanceLookup.new.by_vm_cid(@vm_cid).first
-          @deployment_name = instance.deployment.name
-          @instance_name = instance.name
-          parent_id = add_event
-          @vm_deleter.delete_for_instance(instance, false)
-        rescue InstanceNotFound
-          parent_id = add_event
-          @vm_deleter.delete_vm_by_cid(@vm_cid)
+        event_log_stage = Config.event_log.begin_stage("Delete VM", 1)
+        event_log_stage.advance_and_track(@vm_cid, false) do
```

Comment:
Hi! Sorry this PR has sat for so long. We were looking at this today, but had a question about why the `log_exception` change was necessary. It seems odd for a task to ignore an exception and not consider itself failed. If you recall more about this change, can you clarify for us?

The other changes make sense for better logging to UI about what happened or didn't happen. Just confused why task-level behavior needs to change, too.

Thanks!

Reply:
@barthy1 - wanted to bump this. Were you still looking at removing this logic. Really appreciate the PR. 👍